contents/teams/feature-flags/objectives.mdx

```diff
@@ -1,54 +1,91 @@
-* **Behavioral Cohort Targeting for Messaging** – Enable behavioral targeting in messaging (owner TBD)
+* **Persons Database split completion** – Finish the persons database split from the feature flags perspective (a shared goal with the Ingestion team)
+* **Deprecate /decide** – Remove the `/decide` endpoint from the latest version of the hosted PostHog app while maintaining self-hosted compatibility in the SDKs.
+* **Rewrite /evaluation_reasons** – Modernize `/evaluation_reasons` endpoint to use the Rust-based flag targeting code instead of Django
+* **Behavioral cohort completion** – Finish remaining behavioral cohort targeting work
+
+### Objective 8: Enhanced debugging capabilities <TeamMember name="Dylan Martin" photo />
+Motivation: Improve visibility into flag evaluations for better debugging and troubleshooting, and push these insights to our users wherever possible
 
-**Customer Validation:** Partner with 5-10 key customers to validate use cases, gather feedback, and measure impact on conversion and engagement. This can begin as soon as we have a working POC, potentially in parallel with Stage 2 work. <TeamMember name="Dylan Martin" photo /> and <TeamMember name="Annika Schmid" photo />
+What we will ship:
+* **Enhanced $feature_flag_called** – Include properties used for evaluation in the event
+* **Historical evaluation view** – Show flag evaluation results for a person over time
 
-### Objective 2: Flag Management & Advanced Features  
-Motivation: Solve flag management pain points for large customers and add advanced flag targeting capabilities.
+### Objective 9: Real-time feature flags <TeamMember name="Gustavo Strassburger" photo />
+Motivation: Enable real-time use cases for feature flags, allowing instant updates without page refreshes or app restarts.
 
 What we will ship:
-* **Flag categorization system** – Let customers organize flags into categories and only request the ones they need such, e.g. client flags, server flags, both. <TeamMember name="Phil Haack" photo />
-* **Automatic stale flag pruning** – Build tools to identify and help customers clean up unused/stale flags automatically <TeamMember name="Phil Haack" photo />
-* **Flag rollouts based on other flags** – Enable targeting on flags where flag's evaluation result can determine another's rollout <TeamMember name="Phil Haack" photo />
+* **Real-time flag updates** – Implement WebSocket/SSE-based real-time flag changes
+* **SDK support** – Add real-time capabilities to key SDKs
 
-### Objective 3: Platform Reliability & Scale
-Motivation: Create a highly stable, resilient feature flag system capable of supporting our largest (and soon-to-be largest) customers with enterprise-grade reliability.
+### Objective 10: Remote config as secret store exploration <TeamMember name="Dustin Byrne" photo />
+Motivation: Explore the viability of extending remote config feature flags to serve as a secure configuration and secrets management system.
```

Comment:
this feels like a reach for the quarter IMO – it's extending your hackathon idea but probably not feasible.